update_action.py

```diff
@@ -0,0 +1,42 @@
+"""
+  Scans directories for files that satisfy a specific mask and updates
+  the actions' tags.
+  python3 update_action.py --input “one/*.y*ml two/*.y*ml three/*.y*ml” --name mlflow
+"""
+
+import argparse
+import glob
+import re
+from github import Github
```

Comment:
We don't have this library in the environment, in which you are running this script.
`make setup` in the workflow script does install some dependencies, but not this one.
I would suggest replacing `make setup` with `pip install pygithub==version` [here](https://github.com/neuro-inc/cookiecutter-neuro-project/pull/505/files#diff-b5d2517dc5a22dbb782b9abd6520d8407ec39825213ced57e1bd690f9d15e3fbR18).
WDYT?